### PR TITLE
PLIN-2291 object properties consistent alpha sort order

### PR DESCRIPTION
**Pull request by acofer**
Use fork of json-patch, pass it json-iterator with `SortMapKeys: true`.

### CODE REVIEW

**Comment by acofer** [comment]
Of course! :-) 
I forked *your* fork into `skuid/json-iterator` so that we aren't using a personal repo for a vendored dependency. I merged that feature branch into master in the `skuid` fork.

**Comment by zachelrath** [comment]
Ah, awesome! Didn't realize that. Thanks!

**Comment by acofer** [comment]
Strike the above comment, I should have read more carefully and would have realized you were talking about the `json-iterator/go` fork, not the `json-patch` fork. Yeah, I can merge that in, but first I'll fork it into skuid's github org as well and merge your feature branch for the custom sort function into master.

**Comment by acofer** [comment]
`tis done.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 skuidpages
 .env
 skuid
+!vendor/github.com/skuid
```
Comment (plusplusben) [inline .gitignore]: shouldn't this be skuid-cli?

Comment (plusplusben) [reply .gitignore]: Nevermind. Derp. That's all of the Skuid projects.

**Pull request merged**